packages/patternfly-3/patternfly-react/src/components/Overlay/Overlay.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import { withInfo } from '@storybook/addon-info';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName, DOCUMENTATION_URL, STORYBOOK_CATEGORY } from 'storybook/constants/siteConstants';
+import { name } from '../../../package.json';
+import { MockOverlayManager, basicExampleSource } from './__mocks__/mockOverlayManager';
+import { Overlay } from './index';
+
+const stories = storiesOf(`${storybookPackageName(name)}/${STORYBOOK_CATEGORY.WIDGETS}/Overlay`, module);
+stories.addDecorator(withKnobs);
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Overlay',
+    reactBootstrapDocumentationLink: `${DOCUMENTATION_URL.REACT_BOOTSTRAP_COMPONENT}overlays`
+  })
+);
+
+stories.add(
+  'Overlay story',
+  withInfo({
+    source: false,
+    propTables: [Overlay],
+    propTablesExclude: [MockOverlayManager],
+    text: (
+      <div>
```

Comment:
ditto

Reply:
This is actually consistent with other stories:
https://github.com/patternfly/patternfly-react/blob/aa839a80a005f5d540faeb0e1a0320b7df1cc982/packages/patternfly-3/patternfly-react/src/components/MessageDialog/MessageDialog.stories.js#L65
https://github.com/patternfly/patternfly-react/blob/aa839a80a005f5d540faeb0e1a0320b7df1cc982/packages/patternfly-3/patternfly-react/src/components/Label/Label.stories.js#L39
https://github.com/patternfly/patternfly-react/blob/aa839a80a005f5d540faeb0e1a0320b7df1cc982/packages/patternfly-3/patternfly-react/src/components/Masthead/Masthead.stories.js#L81
https://github.com/patternfly/patternfly-react/blob/aa839a80a005f5d540faeb0e1a0320b7df1cc982/packages/patternfly-3/patternfly-react/src/components/Wizard/Stories/LoadingWizardExampleStory.js#L22

Reply:
oh sorry, I meant using `React.Fragment` here instead of `div`.